BraveRewardsUI/Settings/Tips/Details/TipsDetailViewController.swift

```diff
@@ -182,3 +247,24 @@ extension TipsDetailViewController {
     }
   }
 }
+
+extension TipsDetailViewController {
+  fileprivate func getTipsThisMonth() -> BalanceReportInfo {
+    let components = Calendar(identifier: .gregorian).dateComponents([.month, .year], from: Date())
+    let month = components.month ?? -1
+    let year = components.year ?? 2019
+    return state.ledger.balanceReport(for: ActivityMonth(rawValue: month) ?? .any, year: Int32(year))
+  }
+  
+  fileprivate func setFavicon(url: String, indexPath: IndexPath) {
+    if let faviconURL = URL(string: url) {
+      state.dataSource?.retrieveFavicon(with: faviconURL, completion: {[weak self] favData in
+        if let tableView = (self?.view as? View)?.tableView,
+          tableView.indexPathsForVisibleRows?.contains(indexPath) == true,
+          let cell = tableView.cellForRow(at: indexPath) as? TipsTableCell {
+          cell.siteImageView.image = favData?.image ?? UIImage(frameworkResourceNamed: "defaultFavicon")
```

Comment:
You should check if `favData` is non-nil first and just return early if its nil (since you already set it as defaultFavicon at cell setup)